src/cdk/activation.ts

```diff
@@ -82,16 +80,19 @@ async function registerCdkCommands(explorer: AwsCdkExplorer): Promise<void> {
         command: 'aws.cdk.provideFeedback',
         callback: async () => {
             vscode.env.openExternal(vscode.Uri.parse(cdkProvideFeedbackUrl))
-        }
+        },
+        telemetryName: 'Command_aws.cdk.provideFeedback'
```

Comment:
Should these have the `Command` "namespace"? We were pretty much using those for VS Code-specific constructs, I don't think these will necessarily be VS Code-specific in the long term.